shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/segment/generic/DataTypeLengthSegment.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.sql.parser.sql.segment.generic;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.shardingsphere.sql.parser.sql.segment.SQLSegment;
+
+import java.util.Optional;
+
+@Getter
+@Setter
+public final class DataTypeLengthSegment implements SQLSegment {
+    
+    private int startIndex;
+    
+    private int stopIndex;
+    
+    private int firstNumber;
+    
+    private int secondNumber;
```

Comment:
As I do not think `firstNumber` and `secondNumber` are proper names, i dug into many doc, and found `firstNumber` is supposed to be `precision`; On the other hand, `secondNumber` means `scale`[1]. Can you give them other expressive names?
[1] https://dev.mysql.com/doc/refman/5.7/en/precision-math-decimal-characteristics.html

Reply:
firstLength and secondLength ,is it ok?

Reply:
Actually, `firstxx` and `secondxx` have  no meaning. It is like your name is `Jingshang`,  rather than `firstPerson`. My suggestion is `precision` and `scale`, or `precisionLength` and `scaleLength`. How about you?